src/org/parosproxy/paros/network/HttpHeader.java

```diff
@@ -69,6 +70,7 @@
     public static final String IF_NONE_MATCH = "If-None-Match";
     public static final String USER_AGENT = "User-Agent";
     public static final String ACCEPT_ENCODING = "Accept-Encoding";
+    public static final String ACCEPT = "Accept";

```

Comment:
We should also add a since JavaDoc tag, make it easier to know when it was introduced (similar to the ones in `HttpResponseHeader`, e.g. `CSP`).